bin/oref0-get-profile.js

```diff
@@ -42,40 +42,36 @@ function updatePreferences (prefs) {
 
 if (!module.parent) {
     
-    var cwd = process.cwd()
-        
-    var pumpsettings_input = process.argv.slice(2, 3).pop();
-
-    if (pumpsettings_input !== undefined && pumpsettings_input.indexOf('export-defaults') > 0) {
-      exportDefaults();
-      process.exit(0);
-    }
-
-    if (pumpsettings_input !== undefined && pumpsettings_input.indexOf('update-preferences') > 0) {
-      var prefs_input = process.argv.slice(3, 4).pop();
-	  if (!prefs_input) { console.error('usage: ', process.argv.slice(0, 2), '<preferences.json>'); process.exit(0); }
-      var prefs = require(cwd + '/' + prefs_input);
-      updatePreferences(prefs);
-      process.exit(0);
-    }
-
```

Comment:
This is the part that causes the export-defaults and update-preferences to be missing; the implementation is there but no way to invoke